docs/source/tutorial/setup_analysis_project.rst

```diff
@@ -97,9 +97,9 @@ This measure creates the baseline scenario. Set the following inputs:
   The total number of buildings this sampling is meant to represent. This sets the weighting factors. For the U.S. single-family detached housing stock, this is 80 million homes. 
   
 **Downselect Logic**
-  Logic that specifies the subset of the building stock to be considered in the analysis. Specify one or more ``parameter|option`` as found in the ``resources/options_lookup.tsv``. (This uses the same syntax as the "Apply Upgrade" measure, which is explained in the next section.) For example, if you wanted to only simulate California homes you could enter ``Location Region|CR11`` in this field. (See the entire custom region map `here`_.)
+  Logic that specifies the subset of the building stock to be considered in the analysis. Specify one or more ``parameter|option`` as found in the ``resources/options_lookup.tsv``. (This uses the same syntax as the "Apply Upgrade" measure, which is explained in the next section.) For example, if you wanted to only simulate California homes you could enter ``Location Region|CR11`` in this field. (See the entire `custom region map`_.)
```

Comment:
Couple nitpicks:

which is explained in the next section. <-- add hyperlink?

"...in this field (CR refers to "Custom Region", which based on RECS 2009 reportable domains aggregated into groups with similar climates; see the entire `custom region map`)."